.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea
 .DS_Store
-node_modules
\ No newline at end of file
+node_modules
+dist
+*/package-lock.json
```

Comment:
`package-lock.json` should not be excluded !
This file contains important notes of necessary dependencies